docs/dev/operators.md

```diff
@@ -0,0 +1,62 @@
+# How to add a new operator to the installer?
+
+This document describes how to provide an operator's configuration files and manifests to installer-launched clusters.
+
+One can classify all the manifest/config files of an operator in two categories:
+ - Static
+ - Templatized
+
+The static ones are clearly just byte blobs and the templates are ones that have some variable that needs to be filled up. Typically the variables that need filling up will come from user preferences as specified in the install config (e.g. cluster name, cluster domain, service cidr etc). Other dependencies could be TLS cert keys for example.
+
+## The recommended way
+The static and template files of your operator need to be dealt with separately.
+
+*Static files*
+
+For static files use the Cluster Version Operator (CVO) payload mechanism. There is a particular way to keep the manifest files so that the CVO update payload can pick it up.
+See this document:
+
+https://docs.google.com/document/d/1CGZVEyuloZ9oD4NUArW6dnEpi0WFc6BP2tqPSwFZTCY/edit#heading=h.1zgrwxmpgxbr
```

Comment:
You can replace this reference, and possibly the whole recommended-way section, with a link to [here][1] now that openshift/cluster-version-operator#34 has landed.

[1]: https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/operators.md